.changes/unreleased/Docs-20220728-140258.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Docs
+body: Update dependency inline-source from ^6.1.5 to ^7.2.0
+time: 2022-07-28T14:02:58.441963-07:00
+custom:
+  Author: emmyoop
+  Issue: "5574"
+  PR: "0000"
```

Comment:
Use the PR that has the index file in it.  Otherwise this will be a dead link in the CHANGELOG that get generated.